Dockerfile

```diff
@@ -1,47 +1,43 @@
-FROM node:10.16.0-alpine
+FROM node:12.16.1-alpine AS builder
 
 LABEL maintainer="Andreas Peters <support@aventer.biz>"
 #Upstream URL: https://git.aventer.biz/AVENTER/docker-matrix-dimension
 
-RUN apk add dos2unix --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community/ --allow-untrusted
+WORKDIR /home/node/matrix-dimension
 
-RUN apk update && \
-    apk add --no-cache bash gcc python make g++ sqlite && \
-    mkdir /home/node/.npm-global && \
-    mkdir -p /home/node/app 
+RUN mkdir -p /home/node/matrix-dimension
 
-COPY ./docker-entrypoint.sh /
 COPY . /home/node/matrix-dimension
 
-
-RUN chown -R node:node /home/node/app && \
-    chown -R node:node /home/node/.npm-global && \
-    chown -R node:node /home/node/matrix-dimension
+RUN chown -R node /home/node/matrix-dimension
 
 USER node
 
-ENV PATH=/home/node/.npm-global/bin:$PATH
-ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+RUN npm clean-install && \
+    NODE_ENV=production npm run-script build
 
-RUN cd /home/node/matrix-dimension && \
-    npm install -D wd rimraf webpack webpack-command sqlite3 && \
-    NODE_ENV=production npm run-script build:web && npm run-script build:app 
+FROM node:12.16.1-alpine
 
-USER root
+WORKDIR /home/node/matrix-dimension
+
+COPY ./docker-entrypoint.sh /
 
-RUN apk del gcc make g++ && \
-    rm /home/node/matrix-dimension/Dockerfile && \
-    rm /home/node/matrix-dimension/docker-entrypoint.sh && \
-    dos2unix /docker-entrypoint.sh
```

Comment:
please keep the dos2unix stuff, as it's used to test the image.

Reply:
I have replaced dos2unix by a script to have minimal dependencies to the os of the container.